components/Domain.php

```diff
@@ -23,13 +23,6 @@ class Domain {
          */
         private $domainSingular;
 
-        /**
-         * Name of the related factory
-         *
-         * @var string
-         */
-        private $factoryName;
-
```

Comment:
FactoryName should not be a part of Domain because Domain, as it is meant by its name, contains information about domains. Factories are related to domains but are not considered to be domain strings.